docs/docs/scanner/misconfiguration/custom/index.md

```diff
@@ -120,21 +119,50 @@ Trivy supports extra fields in the `custom` section as described below.
+| schemas.input              | `schema["kubernetes"]`, `schema["dockerfile"]`, `schema["cloud"]` | Used to validate the policy for syntax errors           |  (applied to all input types) | :material-close: | :material-close: |     :material-check:            |
+| scope                      | `package`                                                         | Defines the policy scope                                |  (applied to all input types) | :material-close: | :material-check: |     :material-check:            |
+| custom.id                  | Any characters                                                    | More information provided below                         |              N/A              | :material-close: | :material-check: |     :material-check:            |
+| custom.avd_id              | Any characters                                                    | More information provided below                         |              N/A              | :material-close: | :material-check: |     :material-check:            |
+| custom.severity            | `LOW`, `MEDIUM`, `HIGH`, `CRITICAL`                               | The severity of the misconfiguration                    |            `MEDIUM`           | :material-close: | :material-check: |     :material-check:            |
+| custom.provider            | Any provider available in Trivy                                   | The resource provider this check relates to             |              N/A              | :material-close: | :material-check: |     :material-close:            |
+| custom.service             | The service name of the provider                                  | One of the services available in the provider           |              N/A              | :material-close: | :material-check: |     :material-close:            |
+| custom.short_code          | Any characters                                                    | Descriptive name for the check                          |              N/A              | :material-close: | :material-check: |     :material-check:            |
+| custom.subtypes            | Cloud Resource Subtypes                                           | Refer to the section on subtypes below                  |              N/A              | :material-close: | :material-check: |     :material-close:            |
+| custom.recommended_actions | Any characters                                                    | Describing what the user should do to resolve the issue |              N/A              | :material-close: | :material-check: |     :material-check:            |
+| custom.input.selector.type | Any item(s) in [this list][source-types]                          | More information provided below                         |              N/A              | :material-close: | :material-check: |     :material-check:            |
```

Comment:
We need to update the link to `source-types` to point to [here](https://github.com/aquasecurity/trivy/blob/9361cdb7e28fd304d6fd2a1091feac64a6786672/pkg/iac/types/sources.go#L4)